lib/stripe/billing_portal/session.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Stripe.BillingPortal.Session do
+  @moduledoc """
+  Work with Stripe Billing (aka Self-serve) Portal Session objects.
+
+  You can:
+
+  - Create a session
+
+  Stripe API reference: https://stripe.com/docs/api/self_service_portal
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type t :: %__MODULE__{
+          :id => Stripe.id(),
+          :object => String.t(),
+          :created => Stripe.timestamp(),
+          :customer => Stripe.id() | Stripe.Customer.t() | nil,
+          :livemode => boolean(),
+          :return_url => String.t(),
+          :url => String.t()
+        }
+
+  @type create_params :: %{
+          :customer => String.t(),
+          optional(:return_url) => String.t()
+        }
+
+  defstruct [
+    :id,
+    :object,
+    :created,
+    :customer,
+    :livemode,
+    :return_url,
+    :url
+  ]
+
+  @plural_endpoint "billing_portal/sessions"
+
+  @spec create(create_params, Stripe.options()) :: {:ok, t} | {:error, Stripe.Error.t()}
+  def create(params, opts \\ []) do
+    new_request(opts)
```

Comment:
Looks like the `customer` is required. see https://github.com/code-corps/stripity_stripe/blob/07a71a1fb229f12e8e26da99960eed4d0dc0f9b1/lib/stripe/payment_methods/card.ex#L105

Reply:
👀  good spot thanks, fixed now